stdlib/public/SwiftShims/HeapObject.h

```diff
@@ -161,8 +161,13 @@ static_assert(alignof(HeapObject) == alignof(void*),
   (__swift_uintptr_t) SWIFT_ABI_ARM64_OBJC_RESERVED_BITS_MASK
 #define _swift_abi_ObjCReservedLowBits                                         \
   (unsigned) SWIFT_ABI_ARM64_OBJC_NUM_RESERVED_LOW_BITS
+#if defined(__ANDROID__)
+#define _swift_BridgeObject_TaggedPointerBits                                  \
+  (__swift_uintptr_t) SWIFT_ABI_DEFAULT_BRIDGEOBJECT_TAG_64 >> 8
```

Comment:
Should we make a separate `SWIFT_ABI_DEFAULT_BRIDGEOBJECT_TAG_64` for Android? Something like a `SWIFT_ABI_ANDROID_BRIDGEOBJECT_TAG_64`?

Reply:
We could, but note that this is only shifted for AArch64, so I'd have to define it only for that. I figured shifting it here is the best way to set that up, but I could add another constant instead if you like.